doc/source/examples.rst

```diff
@@ -180,6 +180,59 @@ Unmeasured qubits are ignored by the measurement objects. Also, the
 order that qubits appear in the results is defined by the order the user added
 the measurements and not the qubit ids.
 
+The measurement gate is only used to sample the measured bitstrings and does not
+have  any effect on the state vector. If the user wishes to access the full
+state vector of a circuit that was measured, this is possible using the
+:py:attr:`qibo.base.circuit.BaseCircuit.final_state` property of circuits.
+There reasons that measurement gates do not affect the state are first that
```

Comment:
I think this sentence is hard to understand.

Reply:
I changed this slightly by splitting the hard sentence to smaller ones. Let me know if you believe it is still hard.